app/routes/goal_routes.py

```diff
@@ -0,0 +1,109 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from .routes_helpers import validate_model
+
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+# CREATE ENDPOINT
+@goal_bp.route("", methods=["POST"])
+def create_goal():
+    req_body = request.get_json()
+
+    if "title" in req_body:
+        new_goal = Goal(title=req_body["title"])
+    else:
+        abort(make_response({"details": "Invalid data"}, 400))
```

Comment:
Consider switching this conditional around:
```python
    if "title" not in req_body:
        abort(make_response({"details": "Invalid data"}, 400))
        
    new_goal = Goal(title=req_body["title"])
```

Like this, the guard clause doesn't hide the main focus of the function/route, which is creating a `Goal`, inside an indentation.